plugin/admin/src/components/CMEditViewTranslateLocale/utils/parse-relations.js

```diff
@@ -0,0 +1,70 @@
+import _ from 'lodash'
+import { normalizeRelation } from '@strapi/admin/admin/src/content-manager/components/RelationInputDataManager/utils/normalizeRelations'
```

Comment:
I think this line requires `strapi` to be 4.5.x

This is not documented anywhere and also makes it incompatible with the strapi version on our playground (4.4.5).

Reply:
Ah yes, that is also in the currently latest version of `strapi-plugin-deepl`...

There are two options that I can see:
- Make peer dependency require strapi 4.5
- Copy the normalizeRelation from the original and use it if strapi is at least version 4.5

What would be perfect but does not really work:
- Dynamic import: Webpack can be configured, but everything I tried still required the module to be present at build time. Only option would be to disable webpack completely for the import, but then it might not be possible at all to load the dependency.

WDYT @layaxx?

Reply:
not sure whats best, probably copying normalizeRelation?